src/TimeUtils.h

```diff
@@ -0,0 +1,12 @@
+// Copyright © 2008-2025 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#ifndef _TIMEUTILS_H
+#define _TIMEUTILS_H
+
+namespace TimeUtils {
```

Comment:
This is very hacky and I'm not keen on it, but I also don't like polluting the `Node.h` and every other class just to give it access to a `timeStep` that is either `(1.0 / 60.0)` or calculated only when the `Game` class is instantiated and in use.